webpack/assets/javascripts/react_app/common/IntegrationTestHelper.js

```diff
@@ -55,15 +55,15 @@ export default class IntegrationTestHelper {
     return this.dispatchSpy.mock.calls.filter(isRelevantCall);
   }
   /**
-   * Get the last dispach call
-   * @return {Array} dispach call
+   * Get the last dispatch call
+   * @return {Array} dispatch call
    */
   getLastDispachCall() {
     return this.getDispatchCalls().slice(-1);
   }
   /**
-   * Take a store snapshot
-   * @param  {string} description Snapshoot description
+   * Compare the store with the stored snapshot
+   * @param  {string} description Snapshot description
    */
   takeStoreSnapshot(description) {
```

Comment:
@sharvit by the way, it says the first argument of `toMatchSnapshot` is a [property matcher object](https://jestjs.io/docs/en/snapshot-testing#property-matchers) and not a string, I'm not sure if `description` is doing anything here?

Reply:
Look like it works, see for example:
- https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/integration.test.js#L46
- https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/__snapshots__/integration.test.js.snap#L48

Reply:
@sharvit so it does!  Yay for undocumented API's